src/jti_background.js

```diff
@@ -223,6 +231,34 @@ chrome.storage.sync.get(StorageID, function (templates) {
     }
 });
 
+// Listen for when extension is installed or updated
+chrome.runtime.onInstalled.addListener(
+    function (details) {
+        if (details.reason === 'install' || details.reason === 'update') {
```

Comment:
Just by logging details.reason to console in the Extensions Button Chrome extension. It triggers install and has an 'update extensions' button to trigger update.